src/test/java/com/weborders/step_definitions/Hooks.java

```diff
@@ -2,9 +2,25 @@
 
 import com.weborders.utilities.Driver;
 import io.cucumber.java.After;
+import io.cucumber.java.Before;
+
+import java.util.concurrent.TimeUnit;
 
 public class Hooks {
 
+
+    @Before
+    public void setup (){
+        Driver.get().manage().window().maximize();
+
+        Driver.get();
```

Comment:
We do not driver initialization in here in this project. Why do not you these lines in BeforeMethod in BaseTest class.